### ISSUE TITLE
Clarify use of CF OAuth token for `/encrypt` endpoint

### ISSUE THREAD

**Issue by csterwa** [pivotal-cf/docs-spring-cloud-services#102]
It is difficult to understand that the `/encrypt` endpoint is the only endpoint that uses `cf oauth-token` in the paragraph here:

https://docs.pivotal.io/spring-cloud-services/1-5/service-broker-and-instances.html#get-access-token-for-direct-requests-to-a-service-instance

We should add a section just for `/encrypt` to show how to do shell command with `cf oauth-token` in the authentication for a curl command.

**Comment by fifthposition** [comment]
It seems to me that we do have separate sections for these--there's a "client credentials" section, which is for most endpoints, and a "password credentials" section, which is for `/encrypt`. Definitely we should reorganize this info though. It's much too hard to parse right now.

**Comment by fifthposition** [comment]
This is up at http://docs-pcf-staging.cfapps.io/spring-cloud-services/1-5/service-broker-and-instances.html#get-access-token-for-direct-requests-to-a-service-instance (the section is a partial and can also be accessed at http://docs-pcf-staging.cfapps.io/spring-cloud-services/1-5/common/service-instance-access.html#get-access-token-for-direct-requests-to-a-service-instance).

Also fixed elsewhere in the "Service Broker and Instances" page: the command at Step 3 of http://docs-pcf-staging.cfapps.io/spring-cloud-services/1-5/service-broker-and-instances.html#invoking-actuator-endpoints now has proper formatting.

**Comment by csterwa** [comment]
@fifthposition These changes are great. I do think we need another issue created for moving the `/encrypt` calls to using the CF CLI plugin instead `cf config-server-encrypt` command. I'll do that.
